source/reference/api.txt

```diff
@@ -7,21 +7,21 @@ mongosync API Endpoints
 .. default-domain:: mongodb
 
 :ref:`c2c-api-start`
-   Starts the synchronization between a source and destination cluster.
+   Starts the synchronization session between a source and destination cluster.
 
 :ref:`c2c-api-progress`
-   Returns either an updated status of the synchronization process or an error.
+   Returns either an updated status of the synchronization session or an error.
 
 :ref:`c2c-api-pause`
-   Pauses the current synchronization operation. 
+   Pauses the current synchronization session. 
 
 :ref:`c2c-api-resume`
    Resumes a paused synchronization session based on data stored on the
    destination cluster.
 
 :ref:`c2c-api-commit`
-   Finalizes the sync between the source cluster and the destination
-   cluster and stops continuous sync between clusters.
+   Finalizes the synchronization session between the source cluster and
+   the destination cluster and stops continuous sync between clusters.
 
 :ref:`c2c-api-reverse`
    Reverses the direction of a committed sync operation.
```

Comment:
`operation` -> `session`

Reply:
I believe this one was already changed! Is it not appearing on your end?

Reply:
Oh huh, it wasn't showing up originally, but now my comment is labeled as "outdated". Weird day for Github I guess - you can ignore this 😅